js/assessments/sentenceLengthInTextAssessment.js

```diff
@@ -0,0 +1,61 @@
+var AssessmentResult = require( "../values/AssessmentResult.js" );
+var countTooLongSentences = require( "./checkForTooLongSentences.js" );
+
+/**
+ * Calculates sentence length score
+ * @param {array} sentences The array containing sentences.
+ * @param {object} i18n The object used for translations.
+ * @returns {object} Object containing score and text.
+ */
+var calculateSentenceLengthResult = function( sentences, i18n ) {
+	var recommendedValue = 20;
+	var tooLong = countTooLongSentences( sentences, recommendedValue );
+	var maximumPercentage = 25;
+	var percentage = ( tooLong / sentences.length ) * 100;
+	// Scale percentages from 21.7 to 31.7 to a score. 21.7 scores 9, 31.7 score 3.
+	var unboundedScore = 9 - ( 6 / 10 ) * ( percentage - 21.7 );
```

Comment:
I see this calculation on two places, maybe it worth considering to move this in to a method, or something like that.

Also I would write `6 / 10` as `0.6`.